README.md

```diff
@@ -87,10 +87,10 @@ data that we use to cross check and verify.
 | Tochigi - Utsunomiya City | [宇都宮市における新型コロナウイルス感染症の発生状況](https://www.city.utsunomiya.tochigi.jp/kurashi/kenko/kansensho/etc/1023128.html)
 |Tokushima    | [新型コロナウイルス感染症について](https://www.pref.tokushima.lg.jp/ippannokata/kenko/kansensho/5034012#25)|
 |Tokyo    |[新型コロナウイルスに関連した患者の発生](https://www.bousai.metro.tokyo.lg.jp/taisaku/saigai/1007261/1007509.html)|
-|Tottori    ||
-|Toyama    ||
+|Tottori    |[新型コロナウイルス感染症(COVID-19)特設サイト](https://www.pref.tottori.lg.jp/corona-virus/)|
```

Comment:
Perhaps the page listing tests is a better choice?

- Title: 県内の発生件数、検査件数、相談件数
-  https://www.pref.tottori.lg.jp/289682.htm

They are still at 0 cases, but this link will list all the cases if they appear

Reply:
In general I listed the top level page for Corona Virus information from each prefecture. It sounds like you'd rather have a zoomed in view of the tests? If so I can update.

Reply:
I was thinking that since the table is titled "Prefecture-specific Data", it was worthwhile to link to the page where the officials plan to show the data. The link on the top level page says:

> 本県では今のところ、新型コロナウイルスに関連した患者は発生していません。本県で発生した場合、速やかに「県内の発生件数、検査件数、相談件数」のページで公表します。

The top level page is fine, though, since there are still no cases, and effectively no data.

Reply:
Okay I'll leave as is for now.